app/models/user.rb

```diff
@@ -4,8 +4,8 @@ class User < ApplicationRecord
 
   enum :role, { teacher: "teacher", student: "student" }, scopes: false, default: "student"
   enum :belt, BELTS, scopes: false, default: nil
+  validates :email, :name, :birth_date, presence: true
 
   has_many :groups, dependent: :destroy
-
-  validates :email, :name, :birth_date, presence: true
+  has_many :group_members, through: :group, dependent: :destroy
```

Comment:
`through: :group`  ?

Reply:
@bryanmg 👆🏻
I think from now on you should run a rails console and make sure relationships works as expected **before** `committing`

Reply:
@itsmeurbi I did, and It works well.
Fixed to `:groups` to improve code reading